sw/airborne/modules/computer_vision/lib/vision/demosaic.h

```diff
@@ -0,0 +1,173 @@
+#ifndef DEMOSAIC_H
+#define DEMOSAIC_H
+
+/**
+ * @brief Bilinear demosaicing
```

Comment:
Where is this from? Did you guys write that from scratch?
Just want to make sure you didn't forget some license/copyright header here....